flocker/control/test/test_protocol.py

```diff
@@ -50,6 +54,26 @@
 from .clusterstatetools import advance_some, advance_rest
 
 
+def random_ip():
+    return inet_ntop(AF_INET6, urandom(16))
```

Comment:
See other comment about specifying UUID on a `Node`; I don't think you need this function at all.